files/rd_job_diff.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+
+project="$1"
+job="$2"
+path="$3"
+format="$4"
+
+temp_file=$(mktemp "/tmp/$job.$format.XXXX")
+
+rd jobs list -p "$project" -J "$job" -f "$temp_file" -F $format
+
+diff "$path" "$temp_file" &> /dev/null
+DIFF_STATUS=$?
```

Comment:
We can rely on [cmp(1)](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/cmp.html) to silently compare files instead of generating a diff only to discard it.

```suggestion
cmp -s "$path" "$temp_file"
```